### ISSUE TITLE
delete button? 

### ISSUE THREAD

**Issue by picarica** [jmshrv/finamp#758]
Hello i just wonder is there delet bytton? Like delete from server? I would like to delete some songs and do i need to open jellyfinbapp to delete it? And yes my account is administrator.
And yes i mean delete file from server not when files are downloaded for offline playback

**Comment by Chaphasilor** [comment]
Hey, this is currently not possible. We could probably add this, but hide it behind a setting. I want to be very careful around deleting users' media, hope you understand.

I'll look into adding this together with deleting playlists.

**Comment by picarica** [comment]
> Hey, this is currently not possible. We could probably add this, but hide it behind a setting. I want to be very careful around deleting users' media, hope you understand.
> 
> I'll look into adding this together with deleting playlists.

understantable and reasonable, just wish i could delete redundant media without using jellyfin app, or maybe some trash system would suffice.

**Comment by Chaphasilor** [comment]
Yeah I have a playlist of songs to delete myself, so I know the problem! I'll see what I can do :)

**Comment by NikMAX2077** [comment]
+1. A very much-needed feature.

**Comment by Chaphasilor** [comment]
This will be added in the next beta update!
